docs/personalization/enabling_personalization.md

```diff
@@ -19,18 +20,21 @@ On the welcome screen, enter the following details:
 Select the **I have read and agree to the Terms and Conditions** checkbox, and then click **Submit**.
 Your request is sent to Ibexa, and you will receive a confirmation email in response.
 
-## Configuring customer credentials
+## Set up service parameters
 
-When you receive the confirmation email, ask your administrator to [add the credentials to your configuration](https://doc.ibexa.co/en/latest/guide/personalization/enabling_personalization/#configuring-mandator-credentials).
+When you receive the confirmation email, ask your administrator to:
```

Comment:
Not sure "confirmation email" is the best term here and best fit. 
The customer will receive credentials so maybe we should rephrase it?

Reply:
@dkorostynski 
We've been using this term before, see two lines above. We could generally call it "an email with credentials". WDYS?